organization/_posts/2017-12-13-packaging.md

```diff
@@ -0,0 +1,59 @@
+---
+title: "HSF Packaging Group Meeting #16, December 13, 2017"
+layout: default
+---
+
+# HSF Packaging Group Meeting #16, December 13, 2017
+
+#### *Present*: Graeme Stewart, Ben Morgan, Jakob Blomer, Guilherme Amadio, Patrick Gartung, Javier Cervantes Villanueva, Rafal Pacholek, Chris Green, Pere Mato, Geri Ganis, Rafal, Xavi?, Oana Boeriu
+#### [Indico Agenda and Presentations](https://indico.cern.ch/event/684972/){:target="_hsf_packaging_16"}
+
+## Introduction
+* Updates on [Use Cases document](https://docs.google.com/document/d/1h-r3XPIXXxmr5tThIh6gu6VcXXRhBXtUuOv14ju3oTI/edit), still open for comments and feedback
+* [Draft of CHEP abstract](https://docs.google.com/document/d/1f2NLMOIzvG6Tsvq821eJjWKNtJRb_McpVArUVpOHSbs/edit) presented, feedback by Wednesday 20th December
+
+## Presentation from Guilherme: Gentoo Prefix on CVMFS to Manage HEP Software
+* Overview of Portage, including links to docs
+* Distribution models in HEP
+  * Base Image via binaries, images (automated with Catalyst)
+  * Gentoo Prefix environments - non-root installs, can support macOs
+* Portage for LCG
+  * No setup (inc `LD_LIBRARY_PATH`)
+  * Only dependendency is kernel, single install for all Linux, multiple compilers
+  * Maybe support macOS
+  * Could be deployed through CVMFS
+* Usage examples demonstrated
+  * From CVMFS (from sft.cern.ch repo)
+  * From a busybox container (using host CVMFS)
+  * Gentoo HSF test stack docker container
+* Comments from Patrick on staged installation:
+  * Guilherme: Portage uses `DESTDIR` to stage installs, but can’t move installation from one prefix to another.
+  * For different installation locations need to rebuild (or post-facto hack the `RPATH` setup)
+* Starting the prefix environment boots the built shell, which has a `PATH` to the prefix area; binaries in the prefix area use the prefix dynamic loader, so this is what picks up the requisite prefix libraries (Linux). In this case, uses its own libc. Use of `RUNPATH` also supported on Linux or macOS.
```

Comment:
I am wondering about RUNPATH - is it supported or just RPATH? I will check with Guilherme.